aten/src/ATen/native/cpu/radix_sort.h

```diff
@@ -0,0 +1,196 @@
+#pragma once
+#include <ATen/Config.h>
+
+#if !AT_PARALLEL_OPENMP
+
+namespace at { namespace native {
+
+bool inline is_radix_sort_available() { return false; }
+
+template <typename K, typename V>
+std::pair<K*, V*> radix_sort_parallel(
+    K* inp_key_buf,
+    V* inp_value_buf,
+    K* tmp_key_buf,
+    V* tmp_value_buf,
+    int64_t elements_count,
+    int64_t max_value) {
+  TORCH_CHECK(false, "radix_sort_parallel: ATen is not compiled with OpenMP support");
+}
+
+}} // at::native
```

Comment:
Again, use nested namespaces
```suggestion
} // at::native
```

Reply:
updated!